### DISCUSSION TITLE
Control number of bins in plotAngleDistribution

### DISCUSSION THREAD

**Discussion by nicholasbyres** [mtex-toolbox/mtex#678] [Ask Anything] [answered]
Hello Ralf and Rudiger!

How can I dictate the number of bins in the command plotAngleDistribution?

Many thanks and a Happy New Year!
Nick

**Comment by kilir** [comment, marked as answer]
Hi Nick,
Happy New Year as well.
`'resolution',xy*degree` is the option.

Cheers,
Rüdiger

**Reply by nicholasbyres** [reply]
ah I was missing the *degree! Thanks Rudiger.

Does this need updating? I'm not sure how it all works but happy to help :)
https://mtex-toolbox.github.io/plotAngleDistribution.html

**Comment by ralfHielscher** [comment]
Hi Nick,

yes this indeed needs some love. You can simple do

edit plotAngleDistribution

in but more information into the header. It will then be converted
into an html file.

a happy new year,

Ralf.

********************************************************************
Ralf Hielscher                   Tel: +371-531-38556
Fakultät für Mathematik               +371-531-22200 (Sekr.)
Technische Universität Chemnitz  Fax: +371-531-22109
Reichenhainer Str. 39            E-mail:
ralf.hielscher@mathematik.tu-chemnitz.de
D-09126 Chemnitz                 http://www.tu-chemnitz.de/~rahi
********************************************************************

**Reply by nicholasbyres** [reply]
Hi Ralf,

Sorry I don't follow. Do mean to make changes to the mtex script? Where do I upload this?

Nick

**Comment by ralfHielscher** [comment]
You can also edit it directly online at

https://github.com/mtex-toolbox/mtex/blob/develop/geometry/plotAngleDistribution.m

just press the little pencil

Ralf..



********************************************************************
Ralf Hielscher                   Tel: +371-531-38556
Fakultät für Mathematik               +371-531-22200 (Sekr.)
Technische Universität Chemnitz  Fax: +371-531-22109
Reichenhainer Str. 39            E-mail:
ralf.hielscher@mathematik.tu-chemnitz.de
D-09126 Chemnitz                 http://www.tu-chemnitz.de/~rahi
********************************************************************
